src/modules/keyboardmanager/ui/EditShortcutsWindow.cpp

```diff
@@ -9,9 +9,10 @@ HWND hWndXamlIslandEditShortcutsWindow = nullptr;
 // This variable is used to check if window registration has been done to avoid repeated registration leading to an error.
 bool isEditShortcutsWindowRegistrationCompleted = false;
 
-// Function to create the Edit Shortcuts Window
+ // Function to create the Edit Shortcuts Window
```

Comment:
[nit] It looks like there is an extra space here. You can use the formatter with Ctrl+K D to format the whole file based on the clang file in the repo. That way you don't need to worry about any formatting till when you are ready to push your changes.